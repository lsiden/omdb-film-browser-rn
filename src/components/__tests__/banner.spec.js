import React from "react"
import renderer from "react-native-test-utils"

import Banner from "../banner"

const defaultProps = () => {
  return {
    onPressAbout: () => {}
  }
}

const createWrapper = (props = {}) =>
  renderer(<Banner {...{ ...defaultProps(), ...props }} />)

it("renders", () => {
  const wrapper = createWrapper()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
