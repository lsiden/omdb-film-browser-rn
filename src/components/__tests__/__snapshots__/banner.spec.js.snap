// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#05347f",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginTop": 5,
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "marginBottom": 3,
      }
    }
  >
    Search Open Movie Database
    Â 
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      onPress={[Function]}
      style={
        Object {
          "color": "white",
          "fontSize": 12,
        }
      }
    >
      (OMDB)
    </Text>
  </Text>
</View>
`;
